cockatrice/src/client/ui/picture_loader.cpp

```diff
@@ -175,8 +175,8 @@ void PictureLoaderWorker::processLoadQueue()
         QString cardName = cardBeingLoaded.getCard()->getName();
         QString correctedCardName = cardBeingLoaded.getCard()->getCorrectedName();
 
-        qDebug().nospace() << "PictureLoader: [card: " << cardName << " set: " << setName
-                           << "]: Trying to load picture";
+        //qDebug().nospace() << "PictureLoader: [card: " << cardName << " set: " << setName
```

Comment:
intentional commenting out of debug stuff?

Reply:
Oh, my bad, I'm not sure how this made it into the commit. I temporarily silenced the PictureLoader because, well, it's *noisy* when you load a deck.